docs/source/el7toel8/deprecation.md

```diff
@@ -0,0 +1,27 @@
+# Deprecated functionality in the el7toel8 repository
+
+Here is the list of deprecated functionality in the el7toel8 repository. Each
+version section contains just the newly introduced deprecation. The dates
```

Comment:
Add info about when a deprecated feature will be removed from this list. How long is "newly introduced"?

Reply:
@mportman12  I have used probably wrong wording regarding the chenglish. I mean when a new deprecation is introduced e.g. in v 0.11.0, and it still present (still deprecated) in additional releases (e.g. 0.12.0), it is not listed again. like in case `TMPTargetRepositoriesFacts`

Reply:
“Deprecated functionality is listed under the first version that the functionality is deprecated in. Note that functionality may be deprecated in later versions but are not listed again.”

Reply:
Updated.